Automattic/wp-calypso
# Starter page templates: remove sidebar component

#### Changes proposed in this Pull Request

The “Change Layout” button in the editor provides the same experience as creating a new page, that is, it will change the layout but discard all content (including post title) except for the page slug.

This PR removes the “Change Layout” button.

Say goodbye to it!

**Before**
<img width="336" alt="Screen Shot 2021-01-15 at 5 24 02 pm" src="https://user-images.githubusercontent.com/6458278/104689552-d0fd5780-5756-11eb-8da7-9fe3682ab947.png">

**After**
<img width="557" alt="Screen Shot 2021-01-15 at 5 23 18 pm" src="https://user-images.githubusercontent.com/6458278/104689537-c9d64980-5756-11eb-9820-6cb51d9f3dde.png">


#### Testing instructions

Check out this branch

Run `yarn dev --sync`

Create a new page on a sandboxed site.

Check that the sidebar is Change layout-free.

Find wonder in the small things.

Fixes #48891
